pkg/cmd/cli/describe/projectstatus_test.go

```diff
@@ -214,8 +213,8 @@ func TestProjectStatus(t *testing.T) {
 			Contains: []string{
 				// this makes sure that status knows this can push.  If it fails, there's a "(can't push image)" next to like #8
 				" hours\n  build #7",
-				"on fedora:23",
-				"-> repo-base:latest",
+				"on istag/fedora:23",
+				"-> istag/repo-base:latest",
```

Comment:
I was actually wondering why it wasn't like that before, looking at other test cases.